### PR TITLE
Create README.fa.md

### PR DESCRIPTION
**Pull request by vahidbaghi**
adding persian translation

### CODE REVIEW

File: 1-Introduction/translations/README.fa.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div dir="rtl">
+  
+# مقدمه‌ای بر علم داده
+
+
+![data in action](images/data.jpg)
```
Comment (paladique) [inline 1-Introduction/translations/README.fa.md:6]: this image is currently broken as is as the README file is in a subdirectory, so be sure to update the path

File: 1-Introduction/translations/README.fa.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> تصویر از <a href="https://unsplash.com/@dawson2406?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Stephen Dawson</a> در <a href="https://unsplash.com/s/photos/data?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+شما در این بخش با تعریف علم داده و ملاحظات اخلاقی که یک دانشمند علوم داده باید در نظر داشته باشد آشنا خواهید شد. همچنین با تعریف داده و کمی هم با آمار و احتمالات که پایه و اساس علم داده است آشنا خواهید شد. 
+
+### سرفصل ها
+
+1. [تعریف علم داده](01-defining-data-science/README.md)
```
Comment (paladique) [inline 1-Introduction/translations/README.fa.md:13]: these 4 links are outside of the translations folder and should be updated to the English version path (until we get more Farsi translations 🙂 )

**Review by paladique — CHANGES_REQUESTED** [review]
Thank you for your contribution @vahidbaghi just a few broken link fixes and this will be ready to merge 🎉

File: 1-Introduction/translations/README.fa.md (hunk 2 above)
Comment (vahidbaghi) [reply 1-Introduction/translations/README.fa.md]: I will translate all the texts into Persian soon.

**Review by paladique — APPROVED** [review]

**Pull request merged**